opennms-full-assembly/pom.xml

```diff
@@ -580,6 +580,7 @@
                 <feature>opennms-endpoints-grafana</feature>
                 <feature>opennms-reporting</feature>
                 <feature>opennms-send-event-command</feature>
+                <feature>opennms-search</feature>
```

Comment:
Dependency on the impl. projects should also be added to the pom, otherwise the 🐵 will not be happy.

Reply:
I am not sure what you mean by this.

Reply:
I now know what this means. Will add dependencies to pom.

Reply:
Dependency monkey is happy and jars occur in system directory. All good here.